x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_table.tsx

```diff
@@ -309,22 +307,53 @@ export const ActionsLogTable = memo<ActionsLogTableProps>(
         : [];
     }, [isFlyout, queryParams.withOutputs, withOutputsFromUrl]);
 
+    const actionIdsWithOpenTrays: string[] = useMemo(
```

Comment:
you can just change `getActionIdsWithDetails` from `useCallback` to `useMemo` and then `actionIdsWithOpenTrays` become obsolete

Reply:
465f7dc58abee3bc3ad027602d7995fb1d860d8a